src/VPolygon.jl

```diff
@@ -171,8 +171,7 @@ have been sorted in counter-clockwise fashion.
 In that case a binary search algorithm can be used that runs in ``O(\\log n)``.
 See issue [#40](https://github.com/JuliaReach/LazySets.jl/issues/40).
 """
-function σ(d::AbstractVector{<:Real},
-           P::VPolygon{N})::Vector{N} where {N<:Real}
+function σ(d::AbstractVector{N}, P::VPolygon)::Vector{N} where {N<:Real}
```

Comment:
for the "example" i mean something as simple as

```julia
julia> N = Rational{BigInt}
julia> P = VPolygon([N[1.0, 1.0], N[1.0, -1.0], N[-1.0, -1.0], N[-1.0, 1.0]])

julia> σ(N[1.0, 1.0], P)
2-element Array{Rational{BigInt},1}:
 1//1
 1//1

julia> σ([1.0, 1.0], P)
2-element Array{Float64,1}:
 1.0
 1.0
```

Reply:
BTW, this example would produce an error when the new behavior in #478 is implemented.

Reply:
What exactly would break? Can you try in #480?

Reply:
Ah, now I see. Yes, this should not work, but is this not what we want?

Reply:
yes, this is what we want, i thought we can add a test with the (wrong) usage, or at least mention in the docstring

Reply:
Hm, in which file? This basically holds for every type...

Reply:
True. I thought adding it only on the `VPolygon.jl` and/or hpolygon. But you're right, a section of the manual with some examples would be good. I've opened #483